client/blocks/blog-stickers/index.jsx

```diff
@@ -13,11 +12,10 @@ import QueryBlogStickers from 'components/data/query-blog-stickers';
 import { getReaderTeams, getBlogStickers } from 'state/selectors';
 import BlogStickersList from 'blocks/blog-stickers/list';
 import InfoPopover from 'components/info-popover';
+import { isAutomatticTeamMember } from 'reader/lib/teams';
 
 const BlogStickers = ( { blogId, teams, stickers } ) => {
-	// If the user isn't in the a8c team, don't show the feature
-	const isTeamMember = !! find( teams, [ 'slug', 'a8c' ] );
-
+	const isTeamMember = isAutomatticTeamMember( teams );
 	if ( teams && ! isTeamMember ) {
```

Comment:
It's there so the component still gets rendered if `teams` is null. The call to the `QueryReaderTeams` query component needs to be rendered in order to fetch the teams.

Once we have the teams, it'll be a `[]` if the user isn't a member of any teams.